gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.2.1-all.zip
```

Comment:
I'm not seeing an updated wrapper jar here

Reply:
The _gradle-wrapper.jar_ [checksums](https://gradle.org/release-checksums/) did not change between 5.1 and 5.2.1.